src/scripts/visualizations/tooltip.js

```diff
@@ -38,6 +40,8 @@
             var plotTop = this.options.chart.plotTop;
             var plotHeight = this.options.chart.plotHeight;
             var distance = this.options.tooltip.distance;
+            var distanceX = this.options.tooltip.distanceX ? this.options.tooltip.distanceX : distance;
```

Comment:
@parikhshiv,
At one time, you could overwrite `undefined`, but it hasn't been possible [since ES5](http://es5.github.io/#x15.1.1.3). Besides, the build process (uglify) replaces `undefined` with `void 0`.